src/qiboml/operations/expectation.py

```diff
@@ -56,7 +56,7 @@ def expectation(
         initial_state=initial_state,
         nshots=nshots,
         differentiation_rule=differentiation_rule,
-        exec_backend=exec_backend,
+        # exec_backend=exec_backend,
```

Comment:
This comes from the main, but it fails because there is no `exec_backend` argument, I commented this out here because I am not sure how this is supposed to work, maybe @MatteoRobbiati can provide some insight about this.